CMakeLists.txt

```diff
@@ -61,12 +74,6 @@ if(USE_V3_COMPATIBLE_NAMES)
     add_definitions(-DUSE_BTCPP3_OLD_NAMES)
 endif()
 
-#---- Find other packages ----
-find_package(Threads REQUIRED)
-
-
-set(BEHAVIOR_TREE_LIBRARY ${PROJECT_NAME})
```

Comment:
This was not used anywhere. There is a similar variable `set(BTCPP_LIBRARY ${PROJECT_NAME})`  which is used. I kept that one.